scala/scala.bzl

```diff
@@ -548,6 +581,36 @@ scala_test = rule(
   test=True,
 )
 
+scala_specs2_test = rule(
+  implementation=_scala_specs2_test_impl,
+  attrs={
+     "main_class": attr.string(default="specs2.run"),
+     "_specs2_all": attr.label(default=Label("//specs2:specs2_all"), allow_files=True),
+     } + _implicit_deps + _common_attrs,
+  outputs={
+     "jar": "%{name}.jar",
+     "deploy_jar": "%{name}_deploy.jar",
+     "manifest": "%{name}_MANIFEST.MF",
+     },
+  executable=True,
+  test=True,
+  )
+
+scala_specs2_junit_test = rule(
+  implementation=_scala_specs2_test_impl,
+  attrs={
+     "main_class": attr.string(default="org.junit.runner.JUnitCore"),
+     "_specs2_all": attr.label(default=Label("//specs2:specs2_with_junit"), allow_files=True),
```

Comment:
maybe we can just have a `junit` target, and then in a macro, etc... have junit + specs. Some might just want to run a junit test without any specs.